src/main/java/ligma/Ligma.java

```diff
@@ -0,0 +1,58 @@
+package ligma;
+
+import java.io.IOException;
+import java.time.format.DateTimeParseException;
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+import ligma.command.Command;
+/**
+ * This class represents a Ligma program that keeps track of your to-do list.
+ * Storage object corresponds to the persistent storage
+ * for the Ligma program executed and TaskList object corresponds
+ * to the existing tasks during execution.
+ */
```

Comment:
Overall, I like how all classes are well documented such that they are very to read and understand without spending much time.

Reply:
Thanks for taking the time to review my repo! Much appreciated.